src/software/simulated_tests/robot_enters_region.py

```diff
@@ -8,104 +8,81 @@
 )
 
 
-class RobotEntersRegion(Validation):
+class MinNumberOfRobotsEntersRegion(Validation):
```

Comment:
similarly to this: https://github.com/suchirss/Software/blob/a3a43444fe7b00a546d209cd8bdf703a2c7f07d3/src/software/simulated_tests/ball_enters_region.py#L42

Can you add a function call __repr__ that returns a string of this validation is doing? Basically if the validation fails, it'll print out a helpful message for debugging